io/auth.go

```diff
@@ -91,8 +91,10 @@ func DoRequestUserToAuthenticate() {
 				println("Authentication unsuccessful. Try again later or contact #support on 'https://discord.qovery.com'. ")
 				os.Exit(1)
 			}
+			expiredAt := time.Now().Local().Add(time.Second * time.Duration(30000))
```

Comment:
the expiration is hard coded to 30000 ? What happens if it does change on the auth provider side?